components/backup-plugins/internal/plugins/nats-channels.go

```diff
@@ -0,0 +1,93 @@
+package plugins
+
+import (
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/api/meta"
+
+	"github.com/heptio/velero/pkg/plugin/velero"
+	"github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
+	evclientset "github.com/knative/eventing/pkg/client/clientset/versioned"
+	messagingv1alpha1Client "github.com/knative/eventing/pkg/client/clientset/versioned/typed/messaging/v1alpha1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
```

Comment:
Please group packages per-project, e.g.
```
stdlib...

third-party...

k8s.io/...

knative.dev/... + github.com/knative/...
```
I gives a clearer overview of what the code depends on.